numpy/core/src/umath/absolute.dispatch.cpp

```diff
@@ -0,0 +1,92 @@
+/*@targets
+ * $maxopt $keep_baseline avx512_skx asimd
+ */
+
+#define _UMATHMODULE
+#define _MULTIARRAYMODULE
+#define NPY_NO_DEPRECATED_API NPY_API_VERSION
+
+#define PY_SSIZE_T_CLEAN
+#include <Python.h>
+
+#undef HWY_TARGET_INCLUDE
+#define HWY_TARGET_INCLUDE "absolute.dispatch.cpp"  // this file
+#include <hwy/foreach_target.h>  // must come before highway.h
+#include <hwy/highway.h>
+
+#include "numpy/ndarraytypes.h"
+#include "numpy/npy_common.h"
+#include "numpy/npy_math.h"
+#include "numpy/utils.h"
+
+namespace numpy {
+namespace HWY_NAMESPACE {  // required: unique per target
+
+// Can skip hn:: prefixes if already inside hwy::HWY_NAMESPACE.
+namespace hn = hwy::HWY_NAMESPACE;
+
+// Alternative to per-function HWY_ATTR: see HWY_BEFORE_NAMESPACE
+template <typename T>
+HWY_ATTR void SuperAbsolute(char **args, npy_intp const *dimensions, npy_intp const *steps) {
+  const T* HWY_RESTRICT input_array = (const T*) args[0];
+  T* HWY_RESTRICT output_array = (T*) args[1];
+  const size_t size = dimensions[0];
+  const hn::ScalableTag<T> d;
+  
+  for (size_t i = 0; i < size; i += hn::Lanes(d)) {
+    const auto in = hn::Load(d, input_array + i);
```

Comment:
Are we sure the memory is aligned to the vector size? If not, we should use LoadU/StoreU.

Reply:
Oh, this is worse than that. It should be `LoadN` or `GatherLoadN`. I haven't written the proper load/store logic due to needing the whole set of `GatherLoadN`/`ScatterStoreN` as well.